packages/okta-auth-js/test/spec/mfa-challenge.js

```diff
@@ -1,6 +1,14 @@
 jest.mock('cross-fetch');
 
-var util = require('../util/util');
+var util = require('@okta/test.support/util');
```

Comment:
`util` is imported but not used. is that intentional?

Reply:
It is used in the file, but there is no diff.